tests/kubevirt_helper.go

```diff
@@ -762,3 +763,19 @@ func DeployVMTemplatesAndValidate() error {
 	_, err = task.DoRetryWithTimeout(waitForCompletedAnnotations, importerPodCompletionTimeout, importerPodRetryInterval)
 	return err
 }
+
+func AddToVMNodeMap(vmName, nodeName string, vmNodeMap *map[string]string) {
+	(*vmNodeMap)[vmName] = nodeName
+}
+
+func ValidateAndUpdateVMNode(vmName, nodeName string, vmNodeMap *map[string]string) error {
```

Comment:
nit: rename the function to indicate that it is expecting node to change e.g. ValidateVMNodeChanged

Reply:
Fixed.